### PR TITLE
Update security-api-overview.md

### PR DESCRIPTION
**Pull request by schmurky**
The AH PM informed me of updated info regarding the AH API

### CODE REVIEW

**Comment by schmurky** [comment]
@preetikr hello! I was asked to update this doc as it contains outdated info for advanced hunting API. I am working on getting the edits into M365D AH API doc as well.

File: api-reference/v1.0/resources/security-api-overview.md [hunk 1]
```diff
@@ -28,19 +28,15 @@ Use [runHuntingQuery](../api/security-security-runhuntingquery.md) to run a [Kus
...
+1. Queries explore and return data from the past 30 days.
+2. Results can return up to 100,000 rows.
+3. You can make up to at least 45 calls per minute per tenant. The number of calls varies per tenant based on its size.
+4. Each tenant is allocated CPU resources, based on the tenant size. Queries are blocked if the tenant has reached 100% of the allocated resources until after the next 15-minute cycle. To avoid blocked queries due to excess consumption, follow the guidance in [Optimize your queries to avoid hitting CPU quotas](/microsoft-365/security/defender/advanced-hunting-best-practices). 
+5. If a single request runs for more than three minutes, it times out and returns an error.
+6. A `429` HTTP response code indicates that you've reached the allocated CPU resources, either by number of requests sent, or by allotted running time. Read the response body to understand the limit you have reached. not exceed 124 MB. Exceeding the size limit results in HTTP 400 Bad Request with the message "Query execution has exceeded the allowed result size. Optimize your query by limiting the number of results and try again."
```
Comment (Lauragra) [inline api-reference/v1.0/resources/security-api-overview.md:39]: Please fix the sentence "not exceed 124 MB". I'm not sure if that's stray text that should be deleted?

Comment (schmurky) [reply api-reference/v1.0/resources/security-api-overview.md]: Thank you! Fixed! @Lauragra

**Comment by schmurky** [comment]
This is now ready to merge @Lauragra, changes done.

**Pull request merged**